Makefile

```diff
@@ -472,4 +472,8 @@ dev-release:
 .PHONY: validate-release
 ## validate the operator by installing the releases
 validate-release:
+	$(Q)python3.7 -m venv $(OUTPUT_DIR)/venv3
```

Comment:
do we really need to invoke this particular python version? what if it does not exist on OpenShift CI?

We already have `setup-env` Makefiel rule available, and Ii would move `yq` installation there.

Reply:
I have made `setup-venv` as a depedency. But not moved `yq` installation there. It may not not required in other places.